backend/main.py

```diff
@@ -27,6 +28,10 @@
 app.include_router(member_router, tags=["auth"], prefix="/auth")
 app.include_router(bot_router, tags=["bot"], prefix="/bot")
 
+# app.include_router(member_router, tags=["Auth"], prefix="/auth")
+# app.include_router(bot_router, tags=["Bot"], prefix="/bot")
```

Comment:
why are they commented out? @kelpikz

Reply:
These routes have already been included in the previous line, ig it can be removed

Reply:
These routes have already been included in the previous line, ig it can be removed